admin.php

```diff
@@ -7,39 +7,124 @@
+
+    /**
+     * Template folder directory.
+     */
+    const TEMPLATE_DIR = __DIR__ . '/tpl';
+
+    const TEMPLATE_FILE = 'admin.tpl';
+
+    public function getMenuSort()
+    {
+        return self::MENU_SORT;
     }
 
     /**
-     * @return bool true if only access for superuser, false is for superusers and moderators
+     * Admins or higher can access this plugin.
+     *
+     * You do not need to be a superuser to access this plugin.
      */
-    public function forAdminOnly() {
+    public function forAdminOnly()
+    {
         return true;
     }
 
     /**
-     * Should carry out any processing required by the plugin.
+     * HTML output (gets generated by twig).
      */
-    public function handle() {
+    public function html()
+    {
+        global $conf;
+        $pagesWithoutTitle = PageGetter::getPagesWithoutTitle();
+        $variables = $this->getVariablesForTemplate($pagesWithoutTitle);
+
+        $loader = new Twig_Loader_Filesystem(self::TEMPLATE_DIR);
+        $twig = new Twig_Environment($loader);
+
+        $initLanguage = $this->getLang('');
+
+        $twig->addGlobal('languageText', $this->lang);
```

Comment:
Does this have to be global?

Reply:
Yes because in Twig I am only able to set variables globally or in the templatefile itself. So I chose this way.